en/docs/developer-portal/manage-application.md

```diff
@@ -51,3 +51,62 @@ Let's get started!
 8. Click **:// CURL** to copy the cURL command template for generating the access token. You can add the consumer key and consumer secret to the URL and invoke the token endpoint to generate an API access token.
 
     ![cURL command](../assets/img/developer-portal/manage-applications/curl-command.png){.cInlineImage-half}
+
+
+## Revoke access tokens
+
+In the event of a theft, security violation, or a precaution, an admin can revoke an access token. Choreo provides you with a revoke token endpoint. You can use a utility like a cURL to invoke this endpoint and revoke your access token.  
+
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
+
+    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
+
+2. Click **Applications**.
+3. Click **Production Keys** -> **OAuth 2.0 Tokens** on the left panel.
+4. You will find the **Consumer Key** (client ID), **Consumer Secret** (client secret), **Token Endpoint**, and **Revoke Endpoint** listed here. You can use these values to revoke the access token. 
+
+The parameters required to invoke the following API are as follows:
+
+-   `access_token_to_be_revoked` - The access token to be revoked
+
+-   `<base64 encoded (consumerKey:consumerSecret)>` - Use a base64 encoder to encode your consumer key and consumer secret using the following format. Choreo does not recommend the use of online base64 encoders for this purpose.
+
+- `<consumerKey>:<consumerSecret>` Thereafter, enter the encoded value for this parameter.
+
+    ``` tab="Format"
+        curl -k -v -d "token=<ACCESS_TOKEN_TO_BE_REVOKED>" -H "Authorization: Basic <base64 encoded (consumerKey:consumerSecret)>" -H "Content-Type: application/x-www-form-urlencoded"  https://sts.choreo.dev/oauth2/revoke
+    ```
+    
+    ``` tab="Examples"
+        curl -k -v -d "token=a0d210c7a3de7d548e03f1986e9a5c39" -H "Authorization: Basic OVRRNVJLZWFhVGZGeUpRSkRzam9aZmp4UkhjYTpDZnJ3ZXRual9ZOTdSSzFTZWlWQWx1aXdVVmth" -H "Content-Type: application/x-www-form-urlencoded" ttps://sts.choreo.dev/oauth2/revoke
```

Comment:
Let's use a JWT token instead of an opaque one for the `token` field in the payload